shinydashboard/lantern/modules/contactsmodule.R

```diff
@@ -43,11 +43,11 @@ contactsmodule <- function(
                 mutate(contact_name = ifelse(is.na(contact_name), ifelse(is.na(contact_value), "-", "N/A"), toString(contact_name))) %>%
                 mutate(contact_type = ifelse(is.na(contact_type), "-", toString(contact_type))) %>%
                 mutate(contact_value = ifelse(is.na(contact_value), "-", toString(contact_value))) %>%
-                mutate(condensed_endpoint_names = ifelse(length(strsplit(endpoint_names, ";")[[1]]) > 5, paste0(paste0(head(strsplit(endpoint_names, ";")[[1]], 5), collapse = ";"), "; ", paste0("<a onclick=\"Shiny.setInputValue(\'show_details\',&quot;", endpoint_names, "&quot,{priority: \'event\'});\"> Click For More... </a>")), endpoint_names))
+                mutate(condensed_endpoint_names = ifelse(length(strsplit(endpoint_names, ";")[[1]]) > 5, paste0(paste0(head(strsplit(endpoint_names, ";")[[1]], 5), collapse = ";"), "; ", paste0("<a class=\"lantern-url\" tabindex=\"0\" aria-label=\"Press enter to open a pop up modal containing the endpoint's entire list of API information source names.\" onkeydown = \"javascript:(function(event) { if (event.keyCode === 13){event.target.click()}})(event)\" onclick=\"Shiny.setInputValue(\'show_details\',&quot;", endpoint_names, "&quot,{priority: \'event\'});\"> Click For More... </a>")), endpoint_names))
```

Comment:
On the contacts page, I'm not seeing this. Like I know for NextGen there's a million organization names, and when I look at those endpoints on the Endpoints page, it looks correct, but on this page there's only 0 or 1 organization names listed for the endpoint?

Reply:
also make URLs clickable